linux_sql/scripts/sql/ddl.sql

```diff
@@ -0,0 +1,60 @@
+postgres=# \c host_agent;
+
+-- Create a host_info table if it does not already exist
+CREATE TABLE IF NOT EXISTS PUBLIC.host_info 
+  ( 
+     id               SERIAL NOT NULL, 
+     hostname         VARCHAR NOT NULL, 
+     cpu_number       INT2 NOT NULL, 
+     cpu_architecture VARCHAR NOT NULL, 
+     cpu_model        VARCHAR NOT NULL, 
+     cpu_mhz          FLOAT8 NOT NULL, 
+     l2_cache         INT4 NOT NULL, 
+     "timestamp"      TIMESTAMP NULL, 
+     total_mem        INT4 NULL, 
+     CONSTRAINT host_info_pk PRIMARY KEY (id), 
+     CONSTRAINT host_info_un UNIQUE (hostname) 
+  );
+
+-- Insert three sample data rows
+INSERT INTO host_info (id, hostname, cpu_number, cpu_architecture, cpu_model, cpu_mhz, l2_cache, "timestamp", total_mem) VALUES(1, 'jrvs-remote-desktop-centos7-6.us-central1-a.c.spry-framework-236416.internal', 1, 'x86_64', 'Intel(R) Xeon(R) CPU @ 2.30GHz', 2300, 256, '2019-05-29 17:49:53.000', 601324);
+
+INSERT INTO host_info (id, hostname, cpu_number, cpu_architecture, cpu_model, cpu_mhz, l2_cache, "timestamp", total_mem) VALUES(2, 'noe1', 1, 'x86_64', 'Intel(R) Xeon(R) CPU @ 2.30GHz', 2300, 256, '2019-05-29 17:49:53.000', 601324);
+
+INSERT INTO host_info (id, hostname, cpu_number, cpu_architecture, cpu_model, cpu_mhz, l2_cache, "timestamp", total_mem) VALUES(3, 'noe2', 1, 'x86_64', 'Intel(R) Xeon(R) CPU @ 2.30GHz', 2300, 256, '2019-05-29 17:49:53.000', 601324);
+
+-- Verify inserted data
+SELECT * FROM host_info;
```

Comment:
You probably don't want this in your finished product because it will pollute people's working DBs with dummy data. These lines are for testing only.